code/controllers/configuration.dm

```diff
@@ -32,6 +32,8 @@
 	var/allow_admin_ooccolor = 0		// Allows admins with relevant permissions to have their own ooc colour
 	var/allow_vote_restart = 0 			// allow votes to restart
 	var/allow_vote_mode = 0				// allow votes to change mode
+	var/toggle_maps = 0					// Change from votable maps = 0 to all compiled maps = 1
+	var/toggle_vote_method = 0			// Toggle voting methods: Weighted = 0, Majority = 1
```

Comment:
Looks like these variables are only ever read from, never set

Reply:
These are default values and are set in the voting.js interface

Reply:
That doesn't seem right. Elaborate on that?